drizzle.config.ts

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from 'drizzle-kit';
+import 'dotenv/config';
+
+export default defineConfig({
+  schema: "./src/lib/server/schema/user.ts",
```

Comment:
Better to declare the following as `shema` variable as array of strings. and export a barrel `index.ts` to export all the schemas.

```ts
import { defineConfig } from 'drizzle-kit';
import 'dotenv/config';

export default defineConfig({
	schema: ['./src/lib/server/schemas/'],
	//... other options
});

```